bash_scripting_101/bash_scripting_101.md

```diff
@@ -218,7 +233,7 @@ This character is called a **pipe**, and it's name is a great way to visualize w
 **Proceed with caution!** Using the `Sudo` command is decidedly not best data security practice, but might need to be invoked as you continue to learn bash, as permissioning can be quite granular and dynamic.
 </div>
 
-### Knowledge Check 4
+### Quiz: Permissions
 
 Which numerical sequence grants no restrictions on permissions and should generally be avoided?
```

Comment:
The answer to this is in a link the learner has to click. Since it is important enough to have its own quiz question, let's put that information directly into the previous page.